tests/app/questionnaire/routing/test_operators.py

```diff
@@ -58,14 +58,18 @@ def test_operation_not_equal(operands, expected_result):
     [(2, 1.0), True],
     [(7, 3), True],
     [(next_week, current_date), True],
-    [(next_week, current_date), True],
     # Test False
     [(0.5, 0.7), False],
     [(1.0, 2), False],
     [(3, 7), False],
     [(current_date, next_week), False],
 ]
 
+greater_than_and_less_than_operations_equals = [
```

Comment:
Find the name confusing, isn't this just not_equals?

Reply:
I suppose you could view it either way, its testing to see if the numbers are equal. But its expecting them to not be equal

Reply:
Im happy to change it though if its clearer

Reply:
Although I do see we have an odd way of doing not_equals, which adding this in might make it even more confusing